locales/translation-en.json

```diff
@@ -291,15 +293,18 @@
       "name": "The e-mail address \"@softwerkskammer.org\" where the group can be reached.",
       "title": "Name of the group used in the website, also realname for e-mails sent by the group",
       "x_coord": "Horizontal position on the map",
-      "y_coord": "Vertical position on the map"
+      "y_coord": "Vertical position on the map",
+      "mail_the_contact_persons": "An e-mail button is displayed on the group page, which can be used to write to the contact persons."
```

Comment:
Ich halte die Grammatik für etwas zweifelhaft... "... on the group page. It ..." halte ich für wesentlich glaubwürdiger.